### PR TITLE
Release 168 0

### PR DESCRIPTION
**Pull request by bijayprasadtiwari**

### CODE REVIEW

File: docs/cmdlets-reference/restore-cohesityremotefile.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+### EXAMPLE 1
+```
+Restore-CohesityRemoteFile -TaskName "restore-file-vm" -FileNames /C/data/file.txt -JobId 1234 -SourceId 843 -TargetSourceId 856 -TargetParentSourceId 828 -TargetHostType KWindows -TargetHostCredential (Get-Credential)
+```
+
+Restores the specified file to the target windows VM with the source id 843 from the latest backup.
+
+### EXAMPLE 2
+```
+Restore-CohesityRemoteFile  -FileNames "/C/myFolder" -NewBaseDirectory "C:\temp\restore" -JobId 61592 -SourceId 3517 -TargetSourceId 3098
```
Comment (pyashish) [inline docs/cmdlets-reference/restore-cohesityremotefile.md:29]: Why is the forward and backward slash inconsistent in the same cmdlet ? 

If taking IDs , can you make the example more helpful by getting those cmdlets from other cmdlets and assigning IDs to a variable.

File: src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1 [hunk 2]
```diff
@@ -24,41 +25,42 @@ function Set-CohesityCmdletConfig {
...
-        [Parameter(Mandatory = $false, ParameterSetName = 'RefreshToken')]
-        [ValidateSet($true, $false)]
+        [switch]$LogHeaderDetail = $false,
+        [Parameter(Mandatory = $false)]
         # If set and the token has expired, the framework would attempt refreshing the token.
-        $RefreshToken = $false
+        [switch]$RefreshToken = $false,
+        [Parameter(Mandatory = $false)]
+        # Log file path. For example set the path as C:\temp
+        [string]$LogFilePath
```
Comment (pyashish) [inline src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1:62]: If not specified default to existing behavior.

File: docs/cmdlets-reference/restore-cohesityremotefile.md (hunk 1 above)
Comment (bijayprasadtiwari) [reply docs/cmdlets-reference/restore-cohesityremotefile.md:29]: The payload accepts filenames and newbasedirectory in the given format only.
OK, will add examples, how to extract the IDs.

File: src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1 (hunk 2 above)
Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1:62]: Its been handled in CSLog.

File: docs/cmdlets-reference/restore-cohesityremotefile.md (hunk 1 above)
Comment (bijayprasadtiwari) [reply docs/cmdlets-reference/restore-cohesityremotefile.md:29]: Done. Added example how to extract ids.

**Pull request merged**